pkg/client/fake/client.go

```diff
@@ -27,7 +27,9 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
+	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
Let's add a comment that we are intentionally not using /v5 to use the same dependency as upstream